util/builtinobjects/builtinobjects.go

```diff
@@ -121,6 +128,10 @@ var (
 			{model.BlockContentWidget_CompactList, widget.DefaultWidgetSet, "", false},
 		},
 	}
+
+	// these variables are used to calculate progress bar by goroutine
+	countReader *datacounter.ReaderCounter
```

Comment:
It's seems like not concurrently safe, if user start downloading more 1  archvie

Reply:
Thank you for notice! Excluded package-level varibles